tests/unit/test_masks.py

```diff
@@ -148,11 +159,38 @@ def it_knows_its_mask_tile(self, request):
         tissue_mask_tile_.return_value = expected_mask
 
         tissue_mask = TissueMask()
-
         binary_mask = tissue_mask(tile)
 
         assert binary_mask == expected_mask
 
+    def it_knows_its_mask_tile_and_compute_with_default(self, request):
+        tile = Tile(PILIMG.RGBA_COLOR_500X500_155_249_240, None, None)
+        calculate_tissue_mask_tile_ = method_mock(
+            request, Tile, "calculate_tissue_mask"
+        )
+
+        tissue_mask = TissueMask()
+        _ = tissue_mask(tile)
+
+        calculate_tissue_mask_tile_.assert_called_once()
```

Comment:
the problem of not checking the parameters can hide bugs in this very case: this test is not really checking that the mask is computed with the default filters. The method `Tile.calculate_tissue_mask` could be called:
1. by `Tile.tissue_mask`, with `FiltersComposition(Tile)` as parameter, if the code logic is correct OR
2. by _mask (dispatched to the `Tile`'s method `_`) with `custom_filters`, if the logic is not correct.

So please add the parameters in the assertion :)